themes/jb/layouts/partials/episode/podverseplayer.html

```diff
@@ -1,9 +1,89 @@
 <div class="container">
-    <iframe 
-        id="pv-embed-player" 
-        scrolling="no"
-        src='https://podverse.fm/embed/player?podcastId={{ print .CurrentSection.Params.podverse_podcast_id }}&episodeGuid={{ print .Params.episode_guid}}&episodeMediaUrl={{ .Params.podcast_file }}&episodeTitle={{ .Params.Title }}&episodePubDate={{ .Date.Unix }}'
-        >
-    </iframe>
+  <iframe
+    id="pv-embed-player"
+    scrolling="no"
+    src="https://podverse.fm/embed/player?podcastId={{ print .CurrentSection.Params.podverse_podcast_id }}&episodeGuid={{ print .Params.episode_guid}}&episodeMediaUrl={{ .Params.podcast_file }}&episodeTitle={{ .Params.Title }}&episodePubDate={{ .Date.Unix }}&showALlEpisodes=true"
```

Comment:
It looks like you misspelled (from a camel case perspective (you did `ALl` instead of `All`), I haven't checked yet but did it work?

Reply:
Ooops :)
Good catch. I was testing that param, nothing changed and I forgot to take it out.
I expected to see this difference when [true](https://podverse.fm/embed/player?podcastId=g40Um-HP1&episodeGuid=92ab2f36-1fef-4cbc-8cec-87b245161c1e&episodeMediaUrl=https://aphid.fireside.fm/d/1437767933/f31a453c-fa15-491f-8618-3f71f1d565e5/92ab2f36-1fef-4cbc-8cec-87b245161c1e.mp3&showAllEpisodes=true) and when [false](https://podverse.fm/embed/player?podcastId=g40Um-HP1&episodeGuid=92ab2f36-1fef-4cbc-8cec-87b245161c1e&episodeMediaUrl=https://aphid.fireside.fm/d/1437767933/f31a453c-fa15-491f-8618-3f71f1d565e5/92ab2f36-1fef-4cbc-8cec-87b245161c1e.mp3&showAllEpisodes=false)

Reply:
Fixed [here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/commit/1014c9ca565e0272d509be14fc518c847427cadf)

Thanks :pray: